QuickReference.ipynb

```diff
@@ -19,11 +19,10 @@
     "NASA services can be queried from Python in multiple ways.\n",
     "* Generic Virtual Observatory (VO) queries.\n",
     "  * Call sequence is consistent, including for non-NASA resources.\n",
-    "  * Use the `pyvo` package: https://pyvo.readthedocs.io/en/latest/\n",
-    "  * Known issues/caveats: https://github.com/NASA-NAVO/aas_workshop_2020_winter/blob/master/KNOWN_ISSUES.md\n",
-    "* Astroquery interfaces\n",
+    "  * Use the [`pyvo` package](https://pyvo.readthedocs.io/en/latest/).\n",
+    "  * [Known issues/caveats](https://github.com/NASA-NAVO/aas_workshop_2020_winter/blob/master/KNOWN_ISSUES.md).\n",
+    "* [Astroquery](https://astroquery.readthedocs.io/en/latest/) interfaces:\n",
     "  * Call sequences not quite as consistent, but follow similar patterns.\n",
-    "  * See https://astroquery.readthedocs.io/en/latest/\n",
     "  * Informal Q&A session Tuesday, 5:30pm-6:30pm, NumFocus booth\n",
```

Comment:
Just in reviewing this, I see this oddity, which probably should be removed and dates from an early workshop.  Can you just add this quick change?  

(I mean the line about a session at the NumFocus booth, in case it's not clear)

Reply:
I have removed that line.